### PR TITLE
LISAv2 code cleanup

### PR DESCRIPTION
**Pull request by simonxiaoss**
minor logic change.
1. script sequence change
2. remove low-value tool option
3. rename function name

### CODE REVIEW

File: Run-LisaV2.ps1 [hunk 1]
```diff
@@ -193,19 +188,24 @@ try {
...
-	if ( $CoreCountExceededTimeout ) { $command += " -CoreCountExceededTimeout $CoreCountExceededTimeout" }
-	if ( $TestIterations -gt 1 ) { $command += " -TestIterations $TestIterations" }
+	# Override for TiP #
 	if ( $TiPSessionId) { $command += " -TiPSessionId $TiPSessionId" }
 	if ( $TiPCluster) { $command += " -TiPCluster $TiPCluster" }
-	if ($UseManagedDisks) {	$command += " -UseManagedDisks" }
-	if ($XMLSecretFile) { $command += " -XMLSecretFile '$XMLSecretFile'" }
+	# Attaching Secret file if provided #
+	if ( $XMLSecretFile ) { $command += " -XMLSecretFile '$XMLSecretFile'" }
+
```
Comment (LiliDeng) [inline Run-LisaV2.ps1]: Line has trailing whitespace

**Comment by juhlee-microsoft** [comment]
Code change requires local change-base testing. Look for test results.

File: Run-LisaV2.ps1 (hunk 1 above)
Comment (simonxiaoss) [reply Run-LisaV2.ps1]: fixed.

**Comment by LiliDeng** [comment]
@juhlee-microsoft Paste result here, Simon adjust the code sequence and one method name. Below case contained updated method name, let know if more testing needed.

[LISAv2 Test Results Summary]
Test Run On           : 12/11/2018 06:41:35
ARM Image Under Test  : canonical : ubuntuserver : 18.04-lts : Latest
Total Test Cases      : 1 (1 Pass, 0 Fail, 0 Abort)
Total Time (dd:hh:mm) : 0:0:18
XML File              : TestConfiguration

   ID TestCaseName                                                 TestResult TestDuration(in minutes) 
------------------------------------------------------------------------------------------------------
    1 BVT-VERIFY-DEPLOYMENT-PROVISION                                    PASS                 8.73 
	FirstBoot : PASS 
	FirstBoot : Call Trace Verification : PASS 
	Reboot : PASS 
	Reboot : Call Trace Verification : PASS

**Pull request merged**